wds-hero-widget.php

```diff
@@ -249,8 +252,8 @@ function wds_hero( $args = array(), $content = false ) {
 							$alt = $details['alt'];
 						?>
 
-							<div class="slider">
-								<img src="<?php echo $src; ?>" alt="<?php echo $alt; ?>" />
+							<div class="slider" style="background: url(<?php echo $src; ?>) no-repeat; background-size: cover;">
```

Comment:
Is this change a part of the `id` thing? Wondering what kind of effect this has (I prefer it), but I know I originally built it with `<img>`?